package.json

```diff
@@ -90,6 +90,7 @@
     "redux-form-validators": "^2.1.2",
     "redux-logger": "^2.8.1",
     "redux-thunk": "^2.2.0",
+    "reselect": "^3.0.1",
```

Comment:
@ekohl ack?

Reply:
@ekohl bump?

Reply:
Should be be OK to add. I'll wait with submitting the actual PR because in the past those were open for a very long time and I have closed some because it took too long. Looks like this one is a trivial `./add_npm_package.sh reselect` in our packaging repo.

Reply:
there are multiple PRs that have this dependency, so I would think its safe to add.

Reply:
It won't be used until foreman.spec actually requires it and that's based on package.json so when one of the PRs that adds it is approved I'll open a PR.